docs/resource-sharing/overview.md

```diff
@@ -14,14 +14,16 @@ itself.
 
 There are several ways pilots can join a resource pool:
 
-* Submitted to a batch system through a [*compute entrypoint*](../compute-element/htcondor-ce-overview.md).
-  These jobs are submitted by an external entity, a *pilot factory* based on observed demand in the pool.
-* Sites can launch [*pilot containers*](os-backfill-containers.md) for a specific pool when they have resources they
-  would like to contribute directly.
-* Users can launch *personal pilot containers* within a site's batch system so they can use an existing share or
-  allocation at a site through the open science pool.
-
-The *compute entrypoint* is the most common way pilots are created as it is highly automated and responsive to existing
-demand.
-The site-launched *pilot container* method is useful for to backfill otherwise-idle resources;
-however, at times these pilots may stay idle because there is insufficient demand within the resource pool.
+* Submitted to your local batch system by a [*compute entrypoint*](../compute-element/htcondor-ce-overview.md).
+  These jobs are created by an external entity, a *pilot factory* based on observed demand in the pool. The
+  *compute entrypoint* (CE) is the most common way pilots are created as it is highly automated and responsive
+  to existing demand.
```

Comment:
```suggestion
* Submitted to your local batch system by a [*compute entrypoint*](../compute-element/htcondor-ce-overview.md) (CE).
  These jobs are created by an external entity, a *pilot factory* based on observed demand in the pool.
  The CE is the most common way to receive pilot jobs since they integrate with automated processes that are responsive
  to existing demand.
```